src/mongocxx/test/collection.cpp

```diff
@@ -366,8 +366,8 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
                     // copyIds[idx] doesn't exist, but ids[idx] does.
                     REQUIRE(rhsIdVal != rhs.end());
 
-                    const auto &lhsVal = lhsIdVal.second;
-                    const auto &rhsVal = rhsIdVal->second;
+                    const auto& lhsVal = lhsIdVal.second;
```

Comment:
Lint fixup from prior commit.

Reply:
/me wonders if we should be linting as part of the travis job

Reply:
Yeah... its weird. I really thought we used to?